index.html

```diff
@@ -305,6 +305,50 @@ <h5 class="about-me-heading">About Me</h5>
         </ul>
       </div>
     </section>
-    
+    <footer class="footer">
+      <h2 class="footer-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/xpzbopvv" method="POST">
+        <ul class="contact-section">
+          <li class="form-items">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              class="name"
+              maxlength="20"
+              placeholder="Full name"
+              required
+            />
```

Comment:
- [ ] Please make sure the name field accepts up to 30 characters to follow the requirements. You can change the maxLength value from 20 to 30.